clients/lib/clients/slack/wrapper.rb

```diff
@@ -47,10 +47,6 @@ def handle_name(user_id)
         @all_users ||= slack_web_client.users_list[:members]
       end
 
-      private def all_channels
-        @all_channels ||= slack_web_client.channels_list[:channels]
```

Comment:
the new api is `conversations_list` but it's paginated (I don't think it was limited before). I think it's more efficient to find it by ID rather than getting the whole list then finding it from that list. I've removed this private method since it's no longer used.